packages/flutter_tools/lib/src/device.dart

```diff
@@ -559,7 +559,7 @@ abstract class Device {
         supportIndicator += ' ($type)';
       }
       table.add(<String>[
-        device.name,
```

Comment:
This change doesn't seem to have any associated test changes. If we want this behavior change, we should test it.